reconcile/test/test_ocm_machine_pools.py

```diff
@@ -33,6 +33,7 @@
 
 
 class TestPool(AbstractPool):
+    __test__ = False
```

Comment:
Remove this please, we'll handle it in a different PR

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4041